src/pint/scripts/event_optimize.py

```diff
@@ -241,15 +241,18 @@ def get_fit_keyvals(model, phs=0.0, phserr=0.1):
     fitkeys = [p for p in model.params if not getattr(model, p).frozen]
     fitvals = []
     fiterrs = []
-    for p in fitkeys:
+    fitunits = []
+    for i, p in enumerate(fitkeys):
         fitvals.append(getattr(model, p).value)
-        fiterrs.append(getattr(model, p).uncertainty_value)
+        fitunits.append(getattr(model, p).units)
+        fiterrs.append(float(((getattr(model, p).uncertainty).to(fitunits[i])).value))
```

Comment:
Do you actually need the list of fitunits? It seems like the units are available from `model.PARAM.uncertainty.unit`?

Reply:
I was originally using fitunits for axis labels within the plotting priors method. I found a work around to making a list of fitunits so I can remove that list.

I tried both `model.PARAM.uncertainty.unit` and `model.params.uncertainty` and received the errors: Attribute PARAM not found in TimingModel or any Component and 'list' object has no attribute 'uncertainty'. I am not sure of those are bugs or I made a mistake in loading in the model.

Reply:
Oh, uh, I was taking a shortcut. What I meant was, if your parameter was RAJ, `model.RAJ.uncertainty.unit` or `model.RAJ.units`; these are the same thing you'd get with `getattr(model, "RAJ").uncertainty.unit` or whatever.